R/estimate_truncation.R

```diff
@@ -83,46 +83,8 @@
 #' old_opts <- options()
 #' options(mc.cores = ifelse(interactive(), 4, 1))
 #'
-#' # get example case counts
```

Comment:
Part of the reason to do it like this was to make it crystal clear what the assumed data generating process was. Do you think this just didn't work or if it did work we can instead improve the documentation (or perhaps we already have) to cover this gap?

Reply:
> Do you think this just didn't work or if it did work we can instead improve the documentation

I agree with the original issue that it was best placed elsewhere and linked in the example. I think it's good to keep the example focused without the user having to understand the data generation step. We can link to that as Seb has pointed out in the `./R/data.R` script.

Reply:
Awesome. So action here is to add a link to the new data-raw script